### PR TITLE
docs(readme): contributors list; links to LLM providers

### PR DESCRIPTION
**Pull request by kushal34712**
docs:-Fixed typo , added links for some of them , added contributors list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,14 +52,14 @@
 **🎉 New**:
 Our semantic conventions are now part of OpenTelemetry! Join the [discussion](https://github.com/open-telemetry/community/blob/1c71595874e5d125ca92ec3b0e948c4325161c8a/projects/llm-semconv.md) and help us shape the future of LLM observability.
 
-Looking for the JS/TS version? Check out [OpenLLMetry-JS](https://github.com/traceloop/openllmetry-js).
+Are you looking for the JS/TS version? Check out [OpenLLMetry-JS](https://github.com/traceloop/openllmetry-js).
```
Comment (nirga) [inline README.md:55]: Not needed

**Review by nirga — COMMENTED** [review]
Hey @kushal34712, thanks for this. Please follow the instructions in the PR template - including the name of the PR. I think both PRs can be combined and the contributors should be in the README so they'll be shown at the front of the repo.

**Comment by kushal34712** [comment]
@nirga sir can u tell me which PR should be combined

**Comment by nirga** [comment]
This and #2103

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -3,3 +3,9 @@
 Thanks for taking the time to contribute! 😃 🚀
 
 Please refer to our [Contributing Guide](https://traceloop.com/docs/openllmetry/contributing/overview) for instructions on how to contribute.
+
+## 💫 Contributors
```
Comment (nirga) [inline CONTRIBUTING.md:7]: Can you move this to the readme?

**Comment by kushal34712** [comment]
@nirga sir I have closed #2103 and updated #2102

**Review by nirga — COMMENTED** [review]
Can you not remove the instructions on the PR template , and update the title accordingly? (Should be "docs: ..."

File: CONTRIBUTING.md (hunk 2 above)
Comment (kushal34712) [reply CONTRIBUTING.md]: sir, it's done. Are there any other changes needed?

Comment (nirga) [reply CONTRIBUTING.md]: Yes @kushal34712 see comment above: 

> Can you not remove the instructions on the PR template , and update the title accordingly? (Should be "docs: ..."

Comment (kushal34712) [reply CONTRIBUTING.md]: @nirga sir its done

**Comment by kushal34712** [comment]
sir they are not much relevant but its correct as per the sentence

**Comment by nirga** [comment]
> sir they are not much relevant but its correct as per the sentence

I don't agree, so let's keep it as they were :)

File: README.md [hunk 3]
```diff
@@ -107,44 +107,42 @@ See [our docs](https://traceloop.com/docs/openllmetry/integrations/exporting) fo
...
 
-- ✅ LangChain
+- ✅ [LangChain](https://python.langchain.com/docs/introduction/)
 - ✅ [LlamaIndex](https://docs.llamaindex.ai/en/stable/module_guides/observability/observability.html#openllmetry)
 - ✅ [Haystack](https://haystack.deepset.ai/integrations/traceloop)
 - ✅ [LiteLLM](https://docs.litellm.ai/docs/observability/traceloop_integration)
 
 ## 🌱 Contributing
 
-Whether it's big or small, we love contributions ❤️ Check out our guide to see how to [get started](https://traceloop.com/docs/openllmetry/contributing/overview).
```
[the remark this answers is not in the document]
Comment (kushal34712) [reply README.md:147]: sir I have not found any issues with this it still exists as same as it was

**Comment by kushal34712** [comment]
done sir

**Review by nirga — APPROVED** [review]

**Pull request merged**